ch03.asciidoc

```diff
@@ -386,18 +386,25 @@ This includes which public payment channels are connecting nodes, which capacity
 ==== Onion routing
 
 If the sending node of a payment has selected a path that is supposed to be used to make the payment, the Lightning Network uses an onion routing scheme similar to the famous TOR-network.
-The routing scheme is called the SPHINX mixformat and will be explained in detail in a later chapter.
-For now we want to focus on its properties for the transport of payments which we also call onions.
-
-1. The most important property is that a routing node can only see on which channels it received an onion and on which channel to set up an HTLCs and thus to which peer to forward the onion. This means that no routing node can know who initiated the payment and for whom the payment is supposed to be. The exception of course would be if the node is the recipient. In that case it would know that it was the final destination.
-2. The onions are small enough to fit into a single TCP/IP package and actually even a link layer frame. This will make traffic analysis for intruding the privacy of the payments almost impossible.
-3. The onions are constructed in a way that they will always have the same length independent of the position of the processing node along the path.
-4. Onions can have up to 20 hops included allowing for sufficiently long paths.
-5. The encryption of the onion for every hop uses different ephemeral encryption keys with every single onion. Should a key (in particular the private key of the public node key) leak at some point in time an attacker who collected onions cannot decrypt the other onions that have been stored.
+The routing scheme is called the "SPHINX mixformat" and will be explained in detail in a later chapter.
+For now we want to focus on its properties for the transport of payments.
+A payment package used for routing is called an "onion".
+It is called an "onion" due to the historic use of this word related to the Tor network which is also called the Onion network.
```

Comment:
you explain afterwards how messages are pealed and that they are called onions for that reason. so I am not sure if the historic reason is how we should present this.

Reply:
I do believe that the term comes from Tor and has been used there extensively for 17 years. 
I rephrased because this is more of a side note, it is not crucial to the discussion of LN.
See commit # 3 in a968296e8051c3dfff9f4a9b30403a171d9611c5